src/hotel-management-app/src/app/customer/menu/menu.component.ts

```diff
@@ -22,28 +22,16 @@ export class MenuComponent implements OnInit {
 
   constructor(
     private route: ActivatedRoute, 
-    private http: HttpClient, 
     private cartService: CartService, 
     private restaurantService: RestaurantService) {}
 
   ngOnInit(): void {
     const id = +this.route.snapshot.paramMap.get('id')!;
```

Comment:
instead of assertion operator can we use typecasting (<data> as Type)

Reply:
resolved